main/lint.xml

```diff
@@ -46,8 +46,6 @@
 
     <!-- resources loaded by dynamic name lookup -->
     <issue id="UnusedResources">
-        <ignore path="res/*/cgeo_logo.png" />
-        <ignore path="res/*/cgeo_sendto.png" />
```

Comment:
What does `resources loaded by dynamic name lookup` mean? Could it be, that they are used for our notification system? Just wondering why we have explicitly excluded them from lint...

Reply:
Yes, our notification system may be one reason (as it is with the `attribute_maintenance.png`).
But I've done a text search on all files in `res` and `src` folder and could not find a reference to `cgeo_logo` and `cgeo_sendto` any longer, therefore removed them.

Reply:
Maybe @Lineflyer can help us here? AFAIK the notification system icons aren't defined in-app, but the resource name is provided via the status message.

But you are right, that these old icons should be removed. Maybe we can just rename the "new_icon" resources so that they match the old names...?

Just guessing...

Reply:
The notification system can make use of all png graphics located in `master/main/res/drawable-*` and the name of the resource to use will be send within the reply from the status server to the client, which then uses it.

However we won't use the old logo there I think.

Reply:
> 
> 
> What does `resources loaded by dynamic name lookup` mean? Could it be, that they are used for our notification system? Just wondering why we have explicitly excluded them from lint...

This tries to describe `<issue id="UnusedResources">`.
Without this exclude you have messages for it in the CI. console. (CI dynamic looks for its using)